### PR TITLE
[doc] Add options doc

### PR DESCRIPTION
**Pull request by katyho**
### Summary
- Added a link in the readme to the configuration wiki page. I also modified this page quite a bit - feel free to edit https://github.com/chanzuckerberg/redis-memo/wiki/Configure-RedisMemo
- Added rdoc comments in options class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,18 @@ end
 ```
 Learn more [here](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord).
 
+### Configure RedisMemo
```
Comment (donaldong) [inline README.md:38]: Let's make this an h2 (`##`) and move it right before the `Why RedisMemo?` section.

File: README.md [hunk 2]
```diff
@@ -35,6 +35,18 @@ end
...
+You can configure and set various RedisMemo options in your initializer `config/initializers/redis_memo.rb`:
+```ruby
+RedisMemo.configure do |config|
+
+  config.expires_in = 3.hours
+  config.global_cache_key_version = SecureRandom.uuid
+  ...
```
Comment (donaldong) [inline README.md:45]:
```suggestion
  # ...
```

**Review by donaldong — COMMENTED** [review]
Thanks! Let's mirror the wiki page to the actual source code as rdoc?

File: README.md [hunk 3]
```diff
@@ -35,6 +35,18 @@ end
 ```
 Learn more [here](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord).
 
+### Configure RedisMemo
+You can configure and set various RedisMemo options in your initializer `config/initializers/redis_memo.rb`:
```
Comment (Dingying0410) [inline README.md:39]: nit: 
```suggestion
You can configure  various RedisMemo options in your initializer `config/initializers/redis_memo.rb`:
```

just feeling a bit redundant

**Comment by katyho** [comment]
> Thanks! Let's mirror the wiki page to the actual source code as rdoc?

I generated the rdoc, but is there a good way to embed that html or formatting directly into a github wiki? I searched around a bit and couldn't find much, but maybe did not search hard enough

**Comment by donaldong** [comment]
> is there a good way to embed that html or formatting directly into a github wiki?

We can use the RubyDoc link after a new release!
https://www.rubydoc.info/gems/redis-memo/0.1.4/RedisMemo/Options

File: README.md [hunk 4]
```diff
@@ -102,6 +102,18 @@ end
 ```
 For each `load` call, the cached result on Redis will be used until its dependencies have been invalidated.
 
+## Configure RedisMemo
+You can configure various RedisMemo options in your initializer `config/initializers/redis_memo.rb`:
+```ruby
+RedisMemo.configure do |config|
+
```
Comment (donaldong) [inline README.md:109]: nit: extra line break

**Review by donaldong — APPROVED** [review]
Thanks!

**Pull request merged**